plugins/async-node/core/src/index.test.ts

```diff
@@ -424,30 +521,3 @@ test("replaces async nodes with chained multiNodes singular", async () => {
   expect(view?.actions[1].asset.type).toBe("text");
   expect(view?.actions[2].asset.type).toBe("text");
 });
-
-test("should call onAsyncNode hook when async node is encountered", async () => {
```

Comment:
I don't think this test should be deleted, right?

Reply:
done